src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/GoToDefinitionTests.cs

```diff
@@ -131,31 +131,37 @@ public async Task GoToDefinition_ComponentAttribute_Minimized()
         await TestServices.Editor.WaitForCurrentLineTextAsync("public bool MyProperty { get; set; }", ControlledHangMitigatingCancellationToken);
     }
 
-    [IdeFact(Skip = "https://github.com/dotnet/razor/issues/8036")]
+    [IdeFact]
     public async Task GoToDefinition_ComponentAttribute_InCSharpFile()
     {
         // Create the files
+        const string MyComponentPath = "MyComponent.cs";
         await TestServices.SolutionExplorer.AddFileAsync(RazorProjectConstants.BlazorProjectName,
-            "MyComponent.cs",
+            MyComponentPath,
             """
             using Microsoft.AspNetCore.Components;
             
             namespace BlazorProject;
 
             public class MyComponent : ComponentBase
             {
-                [Parameter] public string MyProperty { get; set; }
+                [Parameter] public string? MyProperty { get; set; }
             }
             """,
             cancellationToken: ControlledHangMitigatingCancellationToken);
 
+        await TestServices.SolutionExplorer.BuildSolutionAndWaitAsync(ControlledHangMitigatingCancellationToken);
+        await TestServices.Workspace.WaitForProjectSystemAsync(ControlledHangMitigatingCancellationToken);
+
+        const string MyPagePath = "MyPage.razor";
         await TestServices.SolutionExplorer.AddFileAsync(RazorProjectConstants.BlazorProjectName,
-            "MyPage.razor",
+            MyPagePath,
             """
             <MyComponent MyProperty="123" />
             """,
             open: true,
             cancellationToken: ControlledHangMitigatingCancellationToken);
+        await TestServices.SolutionExplorer.OpenFileAsync(RazorProjectConstants.BlazorProjectName, MyPagePath, ControlledHangMitigatingCancellationToken);
```

Comment:
Doesn't the `open: true` above do this? Is there an issue with the `AddFileAsync` method we should fix?